### PR TITLE
Support Ownable & ERC1967Admin

### LINKED ISSUE #4: Support Ownable

### LINKED ISSUE #5: Support ERC1967 admin

### PR DESCRIPTION
**Pull request by Amxx**
Fixes #4
Fixes #5

### CODE REVIEW

**Comment by Amxx** [comment]
there is an issue with the paginated getter. Can be reproduced by clicking "more" in the owner of section for 
http://localhost:3000/mainnet/0xc0eb11fbc755D31C6FeCeaAc8760ddCb88c64FE1/account

**Comment by frangio** [comment]
Can you add some more example links in the landing page to contracts that can showcase this new functionality?

File: src/lib/AvailableRoles.svelte [hunk 1]
```diff
@@ -32,11 +32,10 @@
...
-  <TreeList values={tree} />
-{:else if $queryResult.data}
-  <p>Not an AccessControl contract.</p>
-{/if}
-
-<LoadMore store={queryResult} />
+  {#if $queryResult.data?.account?.asAccessControl}
+    <TreeList values={tree} />
+    <LoadMore store={queryResult} />
+  {:else if $queryResult.data}
```
Comment (frangio) [inline src/lib/AvailableRoles.svelte:38]: How does this work? Wouldn't `data` be nullish if the contract is not access control?

File: src/lib/ContractLogs.svelte [hunk 2]
```diff
@@ -41,6 +41,15 @@
     <td class="logs-cell"><EtherscanLink page="tx/{log.transaction.id}"/></td>
   </tr>
   {/if}
+  {#if log.__typename === 'OwnershipTransferred' || log.__typename === 'ERC1967AdminChanged'}
+  <tr>
+    <td class="logs-cell">{log.__typename}</td>
+    <td class="logs-cell"></td>
+    <td class="logs-cell"><Address address={log.owner.id} shorten/></td>
+    <td class="logs-cell"></td>
```
Comment (frangio) [inline src/lib/ContractLogs.svelte:49]: I think we're missing one more column here, and it should probably be the time column.

File: src/lib/Role.svelte [hunk 3]
```diff
@@ -8,11 +8,19 @@
...
   export let address: string;
   export let roleId: string;
+  export let disabled: boolean = false;
 
   $: short = roleId.replace(/^0x(.{4}).*(.{4})$/, '0x$1...$2');
   $: decoded = roleDict[roleId] ?? short;
 </script>
 
-<a href="/{$chain}/role/{address}/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
+<a class:disabled={disabled} href="/{$chain}/{address}/role/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
```
Comment (frangio) [inline src/lib/Role.svelte:17]: This shorter syntax is equivalent:

```suggestion
<a class:disabled href="/{$chain}/{address}/role/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
```

**Pull request merged**